container/root/run.d/10-nginx.sh

```diff
@@ -25,3 +29,9 @@ then
   echo "[nginx] server index is ${SERVER_INDEX}"
   sed -i "s/index index.html index.htm/index ${SERVER_INDEX}/" $CONFIG_SITE
 fi
+
+if [[ $SERVER_GZIP_OPTIONS ]]
```

Comment:
Can you name this `$SERVER_GZIP_ENABLED` or something more binary-ish?  It sounds like there are options contained within the actual variable, but it's behaving like a flag here.